cmd/commands/runtime.go

```diff
@@ -248,26 +248,12 @@ func RunRuntimeList() error {
 	}
 
 	for _, rt := range runtimes {
-		status := "N/A"
-		namespace := "N/A"
-		name := "N/A"
-		cluster := "N/A"
-		version := "N/A"
-		if rt.Status != nil {
-			status = rt.Status.String()
-		}
-		if rt.Namespace != nil {
-			namespace = *rt.Namespace
-		}
-		if rt.ObjectMeta != nil && rt.ObjectMeta.Name != nil {
-			name = *rt.ObjectMeta.Name
-		}
-		if rt.Cluster != nil {
-			cluster = *rt.Cluster
-		}
-		if rt.Version != nil {
-			version = *rt.Version
-		}
+		status := rt.HealthStatus
```

Comment:
about rt.HealthStatus:
I do include it in the query (in go-sdk), and it is part of the Runtime model.
However, the value rt.HealthStatus holds here is nil - and it also doesn't exist on the database.
Shouldn't it exist in the database?
Is there something missing in the installation or in my understanding here?

Reply:
please notice that the current installation process does not include saving HealthStatus in the database